### PR TITLE
Allow to define http port of chat server

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: bin/setenv.sh [hunk 1]
```diff
@@ -124,23 +126,19 @@ else
...
+  # Tomcat HTTP port configuration
+  xmlstarlet ed -L -u '/Server/Service/Connector/@port' -v "${CHAT_HTTP_PORT}" \
+    -u "/Server/@port" -v "${CHAT_SERVER_STOP_PORT}" ${CHAT_APP_DIR}/conf/server.xml || {
+    echo "ERROR during xmlstarlet processing (changing Connector port)"
     exit 1
   }
 
   # JMX configuration
   if [ "${EXO_JMX_ENABLED}" = "true" ]; then
-    # insert the listener before the "Global JNDI resources" line
```
Comment (bourasom) [inline bin/setenv.sh]: The JMX is used by us. Please keep the configuration

Comment (boubaker) [reply bin/setenv.sh]: I know that you use it, in fact this configuration is deprecated in Tomcat and will be removed as [mentioned here](https://tomcat.apache.org/tomcat-8.5-doc/config/listeners.html#JMX_Remote_Lifecycle_Listener_-_org.apache.catalina.mbeans.JmxRemoteLifecycleListener)

In addition, in PRD 6.1 we will upgrade to Tomcat 9, so I will revert this until we 'll do the upgrade in product and let you re-work it.

(Note: The new option to use is `-Dcom.sun.management.jmxremote.rmi.port=XYZ`, see https://tomcat.apache.org/tomcat-9.0-doc/monitoring.html)

Comment (boubaker) [reply bin/setenv.sh]: Done

**Review by bourasom — APPROVED** [review]
LGTM

**Pull request merged**